tests/s390x_tests/consoletest_TOOL_s390_vmconvert.pm

```diff
@@ -34,11 +32,6 @@ sub run {
 
 }
 
-sub post_fail_hook {
-    my $self = shift;
-    #    $self->export_logs();
-}
-
 sub test_flags {
     return {milestone => 1, fatal => 0};
 }
```

Comment:
One last thing is that if they are all milestone, for s390 we don't support snapshots (yet)... I can't correctly recall what happens in the cases where tests fail and milestone = 1, most likely we'll end up with a lot of tests not being executed, if something happens... Any objections against removing the test_flags altogether?

Reply:
Nothing against. But, those modules are always being executed "alone". So, if it fails, it is always the last one.
Maybe that could be improved later. Even if we remove that now, the tests are never executed together with other (just the tipical booting modules)

Reply:
Then I give my blessing :)